Vagrantfile.example

```diff
@@ -30,14 +28,20 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
  # Vagrant can share the source directory using rsync, NFS, or SSHFS (with the vagrant-sshfs
  # plugin). Consult the Vagrant documentation if you do not want to use SSHFS.
  config.vm.synced_folder ".", "/vagrant", disabled: true
- config.vm.synced_folder ".", "/home/vagrant/bodhi", type: "sshfs", sshfs_opts_append: "-o nonempty"
+ config.vm.synced_folder ".", "/home/vagrant/bodhi",
+   ssh_opts_append: "-o IdentitiesOnly=true",
+   sshfs_opts_append: "-o nonempty",
+   type: "sshfs"
```

Comment:
@pypingou Do you want to close this PR, or do you think it should still be modified?